RefRed/sf_calculator/sort_nxsdata.py

```diff
@@ -36,22 +34,13 @@ def __init__(self, arrayOfNXSDataToSort, parent=None):
                 elif _isBeforeSameOrAfter == Position.after:
                     _positionIndexNXSDataToPosition += 1
                 else:
-                    # TODO: it's not clear what this branch does and whether we ever get here
-                    _new_nxsdata = self.mergedNXSData(_nxsdataToPosition, _nxsdataToCompareWith)
-                    _sortedArrayOfNXSData[indexInPlace] = _new_nxsdata
+                    # we should never get here since run number and data (ws) is
```

Comment:
If "we should never get here" is violated, shouldn't we rather throw a warning or error?